client/my-sites/email/titan-mail-quantity-selection/index.jsx

```diff
@@ -287,151 +224,140 @@ class TitanMailQuantitySelection extends React.Component {
-							"we're only charging you for the remainder of the current month.",
-						{
-							args: {
-								price: renewalCost.text,
-							},
-							comment:
-								'%(price)s is a formatted price for a monthly email mailbox subscription, e.g. $3.50, €3.75, or PLN 4.50',
-						}
-				  )
-				: translate(
-						'Your initial purchase price is more than your standard monthly price of %(price)s per mailbox because ' +
-							"we're charging you for the remainder of the current month plus one or more months of service.",
-						{
-							args: {
-								price: renewalCost.text,
-							},
-							comment:
-								'%(price)s is a formatted price for a monthly email mailbox subscription, e.g. $3.50, €3.75, or PLN 4.50',
-						}
-				  );
-
 		return (
-			<React.Fragment>
-				{ pricingTitle }
-				<span>
+			<Notice icon="info-outline" showDismiss={ false } status="is-success">
+				<>
 					{ translate(
-						'Each additional mailbox will cost {{strong}}%(price)s{{/strong}} for this purchase',
+						'You can purchase new mailboxes at the prorated price of {{strong}}%(proratedPrice)s{{/strong}} per mailbox.',
 						{
 							args: {
-								price: purchaseCost.text,
+								proratedPrice: purchaseCost.text,
 							},
 							components: {
 								strong: <strong />,
 							},
 							comment:
-								'%(price)s is a formatted price for each email mailbox, e.g. $3.50, €3.75, or PLN 4.50',
+								'%(proratedPrice)s is a formatted price for a monthly email mailbox subscription (e.g. $3.50, €3.75, or PLN 4.50)',
 						}
-					) }
-				</span>
-				<span>
+					) }{ ' ' }
+					{ purchaseCost.amount < renewalCost.amount
+						? translate(
+								'This is less than the regular price because you are only charged for the remainder of the current month.'
+						  )
+						: translate(
+								'This is more than the regular price because you are charged for the remainder of the current month plus any additional month until renewal.'
+						  ) }{ ' ' }
 					{ translate(
-						'All of your mailboxes are due to renew at {{strong}}%(price)s/mailbox/month{{/strong}} on %(date)s',
+						'All of your mailboxes are due to renew at the regular price of {{strong}}%(fullPrice)s{{/strong}} per mailbox when your subscription renews on {{strong}}%(expiryDate)s{{/strong}}.',
 						{
 							args: {
-								date: moment( expiryDate ).format( 'LL' ),
-								price: renewalCost.text,
+								fullPrice: renewalCost.text,
+								expiryDate: moment( expiryDate ).format( 'LL' ),
 							},
 							components: {
 								strong: <strong />,
 							},
 							comment:
-								'%(price)s is a formatted price for each mailbox, e.g. $3.50, €3.75, or PLN 4.50; ' +
-								'%(date)s is a localized date of the rough form: 17 February 2021',
+								'%(fullPrice)s is a formatted price for a monthly email mailbox subscription (e.g. $3.50, €3.75, or PLN 4.50), %(expiryDate)s is a localized date (e.g. February 17, 2021)',
```

Comment:
First off: why did you shift the comment for `expiryDate` to the same line? It's now harder to see as a developer.

As above, let's drop the "monthly" reference.
```suggestion
								'%(fullPrice)s is a formatted price for an email mailbox subscription (e.g. $3.50, €3.75, or PLN 4.50), %(expiryDate)s is a localized date (e.g. February 17, 2021)',
```

Reply:
> why did you shift the comment for expiryDate to the same line?

I find it easier to read when everything is on the same line 🙂. I'm happy to switch it back though.